lib/libc/Kconfig

```diff
@@ -217,6 +217,17 @@ config MINIMAL_LIBC_TIME
 	  In order to make use of the non-reentrant gmtime(), it is necessary
 	  to set CONFIG_MINIMAL_LIBC_NON_REENTRANT_FUNCTIONS=y.
 
+config MINIMAL_LIBC_STRING_ERROR_TABLE
+	bool "String error table for strerror() and strerror_r()"
```

Comment:
I'd like this to default to 'y' if MINIMAL_LIBC so that apps naively using these APIs will work, and targets needing to save space will be able to trim the strings out by adding a config option.

Reply:
Must have missed that comment when I scanned over this issue.